addons/mail/static/src/messaging/component/composer/composer.js

```diff
@@ -225,7 +225,7 @@ class Composer extends Component {
 
         const context = {
             // default_parent_id: this.id,
-            default_body: mailUtils.escapeAndCompactTextContent(this._textInputRef.comp.getContent()),
+            default_body: mailUtils.escapeAndCompactTextContent(this.composer.textInputContent),
```

Comment:
Just to make sure: will this get the most recent content? 
Save input content to composer entity is correctly called before?